apps/etl-func/src/domain/message-status.ts

```diff
@@ -25,25 +25,52 @@ const statusEnum = z.enum([
 
 const rejectedMessageStatusSchema = z.object({
   fiscalCode: fiscalCodeSchema,
+  id: z.string().min(1),
   isArchived: z.boolean().default(false),
   isRead: z.boolean().default(false),
   messageId: z.string().ulid(),
   rejection_reason: rejectionReasonSchema,
   status: statusEnum.extract(["REJECTED"]),
   updatedAt: z.number(),
+  version: z.number().gte(0),
 });
 
 const notRejectedMessageStatusSchema = z.object({
   fiscalCode: fiscalCodeSchema,
+  id: z.string().min(1),
   isArchived: z.boolean().default(false),
   isRead: z.boolean().default(false),
   messageId: z.string().ulid(),
   status: statusEnum.exclude(["REJECTED"]),
   updatedAt: z.number(),
+  version: z.number().gte(0),
 });
 
 export const messageStatusSchema = z.discriminatedUnion("status", [
```

Comment:
if there are common property they can be declared once in `messageStatusSchema`

Reply:
[25c257c](https://github.com/pagopa/io-messages/pull/62/commits/25c257c5e1efc5842d8ebb931faf087cecf68d56)